lib/java/persistance/src/main/java/org/enso/persist/PerBufferReference.java

```diff
@@ -1,21 +1,28 @@
 package org.enso.persist;
 
+import java.io.IOException;
 import org.enso.persist.PerInputImpl.InputCache;
 import org.enso.persist.Persistance.Reference;
 
 final class PerBufferReference<T> extends Persistance.Reference<T> {
   private final Persistance<T> p;
   private final PerInputImpl.InputCache cache;
   private final int offset;
+  private Object cached;
```

Comment:
What is the meaning of the `cached` field? I'm not sure if I understand it.

Reply:
It is a tri-state value. Explained in 0b09978841